src/main/user-impl/java/com/mysql/cj/jdbc/ha/util/RdsUtils.java

```diff
@@ -124,6 +153,22 @@ public String getRdsInstanceHostPattern(final String host) {
     return "?";
   }
 
+  public String getRdsClusterHostUrl(final String host) {
```

Comment:
all those functions could be `static` - what do you think?

Reply:
done